### PR TITLE
win 531.29 dch

### PR DESCRIPTION
**Pull request by OMEGARAZER**
**Purpose of proposed changes**

Adds Windows 531.29 DCH patches

### CODE REVIEW

**Comment by JShiesty-dev** [comment]
nvencodeapi64 results in "problem patching".

File: drivers.json [hunk 1]
```diff
@@ -3740,9 +3740,18 @@
                     "product": "GeForce",
                     "version": "531.26",
                     "variant": "DCH (Hotfix)",
-                    "patch64_url": "win10_x64/531.18/nvencodeapi64.1337",
```
Comment (jailuthra) [inline drivers.json:3743]: Oops missed this in the last PR. I hope you're not manually editing these, there are helper scripts to do all of this :)

**Review by jailuthra — APPROVED** [review]
Thanks lgtm.

File: drivers.json (hunk 1 above)
Comment (OMEGARAZER) [reply drivers.json:3743]: I did in the last one and missed my mistake.

**Pull request merged**